source/whats-new.txt

```diff
@@ -12,10 +12,24 @@ What's New
 
 Learn what's new in:
 
+* :ref:`Version 1.3 <mongodb-analyzer-whats-new-1.3>`
 * :ref:`Version 1.2 <mongodb-analyzer-whats-new-1.2>`
 * :ref:`Version 1.1 <mongodb-analyzer-whats-new-1.1>`
 * :ref:`Version 1.0 <mongodb-analyzer-whats-new-1.0>`
 
+.. _mongodb-analyzer-whats-new-1.3:
+
+What's New in 1.3
+-----------------
+
+New features of the 1.3 {+product+} release include:
+
+- POCO to JSON analysis rules that show the serialization result of a POCO at compile
+  time.
+- POCO, LINQ, and Builders analysis support BSON serialization attributes.
+- Support for LINQ expressions that use enums in tuples. 
+- Improved support for nullable types.
```

Comment:
Suggestion:

I think all the list items are these are fragments rather than full sentences (perhaps the last one could be read either way), so they should not have periods at the end.

From the style guide on [list items](https://www.mongodb.com/docs/meta/style-guide/style/lists/#list-items):
>    -  In a list of only fragments, use no punctuation at the end of each item.

Reply:
The second item is a full sentence.

Reply:
In that case, I would recommend changing all of them to be sentences to match this guidance from the same spot in the style guide:

> Make all list items parallel. For example, all items start with fragments, or all items use sentences. A list can have a mix of fragments and sentences as long as all of the items start with a fragment.